front-end-web/src/Components/Pages/ExportPage.tsx

```diff
@@ -1,16 +1,28 @@
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
+import { Button, TextField } from "@material-ui/core";
 
 /**
  * TODO: Add Documentation
  */
+
 const ExportPage: React.FunctionComponent<Props> = ({
   state,
   handlers,
   classes
 }) => {
+  document.cookie = "stage=4";
   return (
     <Fragment>
+      <Button
+        variant="contained"
+        onClick={function() {
+          document.cookie = "stage=4";
+        }}
+      >
+        Export
+      </Button>
+      <TextField variant="standard" label="Calendar name"></TextField>
```

Comment:
Not super important, but if we could change this to full width and outlined, that would be great!